cmd/svcat/output/yaml.go

```diff
@@ -59,3 +60,16 @@ func writeParameters(w io.Writer, parameters *runtime.RawExtension) {
 		writeYAML(w, params, 2)
 	}
 }
+
+func writeParametersFrom(w io.Writer, parametersFrom []v1beta1.ParametersFromSource) {
+	if parametersFrom == nil || len(parametersFrom) == 0 {
```

Comment:
Type safety combined with passing by value means that the first part of the if is unnecessary.